eip-0034.md

```diff
@@ -0,0 +1,121 @@
+# NFT Collection Standard V1
+
+* Author: anon_real
+* Status: Proposed
+* Created: 10-May-2022
+* License: CC0
+* Forking: not needed 
+
+## Motivation 
+NFT collections play a central role in the nowadays NFT ecosystem on all blockchains. It gives the ability to artists to organize their artworks and present them in a more meaningful way. It also can facilitate the use of other NFT features such as traits. As we'll present in this proposal, it can also enable more advanced features such as artist collaborations. These advanced features can get arbitrary complex with the use of smart contracts.
+
+## Design
+We consider collection NFTs as tokens! The issued collection token will contain all the collection information.
+
+An NFT will be considered as part of an already created collection if it follows the [artwork standard]() and the collection NFT is present in the first input of the issuance transaction which is known as the royalty box.
+
+There is no limitation on the number of issued tokens for a collection. One possible interpretation of such a feature is that artists can come together and create a collection. Each one of them will have one collection token which gives the right to them to add NFTs to that collection. We believe that with the use of smart contracts, more advanced usage of this flexibility will be discovered in the future.
```

Comment:
Some pre-defined limitations protected in SC would be a cool feature actually. 
collection NFT mint power expiration date - can't mint more NFT for that collection after a specific date

Reply:
Protection is SC can not happen as tokens can not be necessarily tied to SCs. But having expiry data was part of the design which is not pushed yet. I'll push that soon as well.
This expiry date can be enforced by marketplaces.